### PR TITLE
channel: set write buffer size

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

File: src/channel.rs [hunk 1]
```diff
@@ -188,6 +189,12 @@ impl ChannelBuilder {
         self
     }
 
+    pub fn http2_write_buffer_size(mut self, size: usize) -> ChannelBuilder {
```
Comment (siddontang) [inline src/channel.rs]: add comment

**Comment by disksing** [comment]
LGTM.

**Comment by BusyJay** [comment]
@sre-bot r+

**Comment by BusyJay** [comment]
@sre-bot retry

**Pull request merged**